packages/veritone-widgets/src/widgets/SourceManagementForm/story.js

```diff
@@ -0,0 +1,282 @@
+import React from 'react';
```

Comment:
Ideally this should be refactored to use <BaseStory>

Reply:
you didn't finish your thought here

Reply:
to use the BaseStory component